db/migrate/20220326181536_users.rb

```diff
@@ -0,0 +1,11 @@
+class Users < ActiveRecord::Migration[7.0]
+  def change
+    create_table :Users do |t|
+      t.string :Name
+      t.string :Photo
+      t.text :Bio
+      t.timestamps
+      t.integer :PostsCounter
```

Comment:
- [ ] <b>REQUIRED:</b> In rails, it is much better to use `lower case` and `snake case` when naming. For example in this case, `Name` should be `name`,  `Photo` should be `photo`,  `Bio ` should be `bio `,  and `Post_Counter ` should be `post_counter `.